File: pkg/definitions/updaters/python/fixtures/invalid_code_out_airplane.py
# This file includes a few intentional "bugs" to ensure the update gracefully
# handles them.
import airplane

# Bug: `airplane` was not imported

# Bug: unknown function reference
ipsum()


@airplane.task(description="Added a description!")
def my_task():
    # Bug: unknown variable reference(foo).
    print(foo)

File: pkg/definitions/updaters/javascript/fixtures/invalid_code.out.airplane.js
// This file includes a few intentional "bugs" to ensure the update gracefully
// handles them.

// Bug: `airplane` is not imported.

// Bug: unknown function reference
ipsum()

export default airplane.task(
  {
    slug: "my_task",
    description: "Added a description!"
  },
  async () => {
    // Bug: unknown variable reference (foo).
    console.log(foo)
  }
);
